### DISCUSSION TITLE
不懂就问，v2下载接下来应该进行什么操作

### DISCUSSION THREAD

**Discussion by DDDragonL** [the1812/Bilibili-Evolved#2515] [提问 / Q&A] [answered]
![螢幕擷取畫面 2021-11-14 091129](https://user-images.githubusercontent.com/94264798/141663802-8f6a9d18-080f-4017-949e-8c4a566d0ef1.png)

**Comment by the1812** [comment, marked as answer]
点开始啊
![image](https://user-images.githubusercontent.com/26504152/141960776-6ec9fc1d-f290-44d0-8470-815653af76d6.png)

**Reply by DDDragonL** [reply]
但是我截图里面已经截到最低端了，并没有开始按钮

**Reply by the1812** [reply]
那你贴下控制台输出? Ctrl + Shift + I

**Reply by DDDragonL** [reply]
![QQ截图20211120155420](https://user-images.githubusercontent.com/94264798/142719054-2524f3e7-dc42-4d91-aa19-abce089eee5f.png)
是这个吗

**Reply by the1812** [reply]
是的, 开着下载视频面板的时候, 往这里面输入 `document.querySelector('.download-video-panel-footer')?.outerHTML` 运行下看看
![image](https://user-images.githubusercontent.com/26504152/142720567-75467cd4-ffd9-4e10-8614-56cb3438e518.png)

**Reply by DDDragonL** [reply]
似乎不起作用
